frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -607,7 +607,7 @@ export const dashboardLogic = kea<dashboardLogicType>([
                     ? JSON.stringify(
                           {
```

Comment:
Do we have a type for this to do `as DashboardTemplate` inside `JSON.stringify()`, to make sure this doesn't get out of sync?

Reply:
👍 if we don't I can add one...

I want to be able to validate this in the template repository too